v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,337 @@
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    <tr>
+      <td style="color:#46a417"><b>AZ/Rack</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>Region</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>Region + 1 Node</b></td>
+      <td>Rep Factor = 7</td>
+      <td>Rep Factor = 7</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 Nodes</b></td>
+      <td>Rep Factor = 5</td>
+      <td>Rep Factor = 5</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 Regions</b></td>
+      <td>Not possible</td>
+      <td>Not possible</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 Regions + 1 Node</b></td>
+      <td>Not possible</td>
+      <td>Not possible</td>
+      <td>Rep Factor = 11</td>
+    </tr>
+  </tbody>
+</table>
+
+For example, a 3-region, 9-node cluster would need a minimum replication factor of 3 to survive a disk failure.
+
+### Multi-region recovery
+
+For hardware failures in a multi-region cluster, the actions taken to recover vary and depend on the type of infrastructure used (e.g., virtual machines, Kubernetes, bare metal).
+
+For example, consider a CockroachBD cluster with the following setup:
+
+- Multi-region
+- 9 nodes
+- Each node is in a separate availability zone
+- 3 nodes in each region
+- Replication factor of 3
+
+The table below describes what actions to take to recover from various hardware failures in this example cluster:
+
+<table>
+  <thead>
+    <tr>
+      <th>Simultaneous Failure</th>
+      <th>Availability</th>
+      <th>Consequence</th>
+      <th>Action to Take</th>
+    </tr>
+  </thead>
+  <tbody class=nohover>
+    <tr>
+      <td style="color:#46a417"><b>1 Disk</td>
+      <td style="color:#228B22"><b>√</b></td>
+      <td rowspan="5">Under-replicated data. Less resources for workload.</td>
+      <td><a href="start-a-node.html">Restart the node</a> with a new disk.</td>
+    </tr>
+      <td style="color:#46a417"><b>1 Server</td>
+      <td style="color:#228B22"><b>√</b></td>
+      <td rowspan="3">If using a Kubenetes StatefulSet, a node will be restarted automatically. <br><br>If the server, rack, or AZ becomes available check the <a href="admin-ui-overview-dashboard.html">Overview dashboard</a> on the Admin UI:
+      <br><br>- If the down server is marked <b>Suspect</b>, try <a href="start-a-node.html">restarting the node</a>.
+      <br>- If the down server is marked <b>Dead</b>, <a href="remove-nodes.html">decommission the node</a> and add a new server. If you try to rejoin the same decommissioned node back into the server, you should wipe the store path before rejoining.</td>
```

Comment:
Should we add a note here about ensuring locality flags are set correctly on node startup? Or is that a given?